README.md

```diff
@@ -152,25 +152,18 @@ requests to the backend for execution and the backend uses the API to
 communicate with Triton.
 
 Every model must be associated with a backend. A model's backend is
-specified in the model's configuration using the 'backend' and
-'platform' settings. Depending on the backend one or the other of
-these properties is optional.
+specified in the model's configuration using the 'backend' setting. 
 
-* For TensorRT, 'backend' must be set to *tensorrt* or 'platform' must
-  be set to *tensorrt_plan*.
+* For TensorRT, 'backend' must be set to *tensorrt*.
 
-* For PyTorch, 'backend' must be set to *pytorch* or 'platform' must
-  be set to *pytorch_libtorch*.
+* For PyTorch, 'backend' must be set to *pytorch*.
 
-* For ONNX, 'backend' must be set to *onnxruntime* or 'platform' must
-  be set to *onnxruntime_onnx*.
+* For ONNX, 'backend' must be set to *onnxruntime*.
 
-* For TensorFlow, 'platform must be set to *tensorflow_graphdef* or
-  *tensorflow_savedmodel*. Optionally 'backend' can be set to
-  *tensorflow*.
+* For TensorFlow, 'backend' must be set to *tensorflow*.
```

Comment:
`For <>, `backend` must be set to <> ` format is repeated 4 times. I think the `backend` is set to the name of the backend for all backends. I don't see much value in listing it like this. How about?

==>

 specified in the model's configuration using the 'backend' setting.  For using TensorRT backend, the value of this setting should be *tensorrt*. Similarly, for using PyTorch, ONNX and TensorFlow Backends, the `backend` field should be set to *pytorch*, *onnxruntime* or *tensorflow* respectively.

Reply:
Fixed.